client/concordclient/include/client/concordclient/concord_client.hpp

```diff
@@ -18,7 +20,7 @@
 #include "bftclient/base_types.h"
 #include "bftclient/bft_client.h"
 
-namespace client::concordclient {
+namespace concord::client::concordclient {
```

Comment:
The repetition here makes me itchy... Same for the path. I'm not sure exactly what to do about it though.

Reply:
I feel similar but the repetition comes from the directory structure. I couldn't think of a better name.
However, I feel strongly that `client` shouldn't be the root namespace but the project name instead.
`concord::ConcordClient` would be better but I'm trying to be consistent with the directory structure and include paths.

Reply:
I feel similar. I was guided by two principals. First, the root namespace should be the project name or something more meaningful than `client`. Second, the namespace hierarchy should follow the directory structure, similar to include paths. I don't have a better suggestion but I'm feeling the itch too :/